### PR TITLE
CTE AWS GuardDuty 1.0.0 - Initial release

### PR DESCRIPTION
**Pull request by nijrajgelani-crest**

### CODE REVIEW

File: aws_guardduty/lib/botocore/data/dms/2016-01-01/examples-1.json [hunk 1]
```diff
@@ -0,0 +1,1074 @@
...
+      {
+        "input": {
+          "CertificateArn": "",
+          "DatabaseName": "testdb",
+          "EndpointIdentifier": "test-endpoint-1",
+          "EndpointType": "source",
+          "EngineName": "mysql",
+          "ExtraConnectionAttributes": "",
+          "KmsKeyId": "arn:aws:kms:us-east-1:123456789012:key/4c1731d6-5435-ed4d-be13-d53411a7cfbd",
+          "Password": "pasword",
```
Comment (ns-snekkar) [inline aws_guardduty/lib/botocore/data/dms/2016-01-01/examples-1.json:40]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: aws_guardduty/lib/botocore/data/eks/2017-11-01/examples-1.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
+{
+  "version": "1.0",
+  "examples": {
+    "CreateCluster": [
+      {
+        "input": {
+          "version": "1.10",
+          "name": "prod",
+          "clientRequestToken": "1d2129a1-3d38-460a-9756-e5b91fddb951",
```
Comment (ns-snekkar) [inline aws_guardduty/lib/botocore/data/eks/2017-11-01/examples-1.json:9]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: aws_guardduty/lib/botocore/data/fsx/2018-03-01/examples-1.json [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+        },
+        "description": "This operation creates a new backup.",
+        "id": "to-create-a-new-backup-1481840798597",
+        "title": "To create a new backup"
+      }
+    ],
+    "CreateFileSystem": [
+      {
+        "input": {
+          "ClientRequestToken": "a8ca07e4-61ec-4399-99f4-19853801bcd5",
```
Comment (ns-snekkar) [inline aws_guardduty/lib/botocore/data/fsx/2018-03-01/examples-1.json:95]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: aws_guardduty/lib/botocore/data/fsx/2018-03-01/examples-1.json [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+        "description": "This operation creates a new Amazon FSx for Windows File Server file system.",
+        "id": "to-create-a-new-file-system-1481840798547",
+        "title": "To create a new file system"
+      }
+    ],
+    "CreateFileSystemFromBackup": [
+      {
+        "input": {
+          "BackupId": "backup-03e3c82e0183b7b6b",
+          "ClientRequestToken": "f4c94ed7-238d-4c46-93db-48cd62ec33b7",
```
Comment (ns-snekkar) [inline aws_guardduty/lib/botocore/data/fsx/2018-03-01/examples-1.json:170]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: aws_guardduty/lib/botocore/data/importexport/2010-06-01/paginators-1.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+  "pagination": {
+    "ListJobs": {
+      "input_token": "Marker",
```
Comment (ns-snekkar) [inline aws_guardduty/lib/botocore/data/importexport/2010-06-01/paginators-1.json:4]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: aws_guardduty/lib/botocore/data/importexport/2010-06-01/paginators-1.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+      "output_token": "Jobs[-1].JobId",
```
Comment (ns-snekkar) [inline aws_guardduty/lib/botocore/data/importexport/2010-06-01/paginators-1.json:5]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: aws_guardduty/lib/botocore/data/s3/2006-03-01/examples-1.json [hunk 7]
```diff
@@ -0,0 +1,1843 @@
...
+              "LastModified": "2014-05-02T04:51:50.000Z",
+              "Size": 4192256,
+              "StorageClass": "STANDARD"
+            }
+          ],
+          "IsTruncated": true,
+          "KeyCount": "2",
+          "MaxKeys": "2",
+          "Name": "examplebucket",
+          "NextContinuationToken": "1w41l63U0xa8q7smH50vCxyTQqdxo69O3EmK28Bi5PcROI4wI/EyIJg==",
```
Comment (ns-snekkar) [inline aws_guardduty/lib/botocore/data/s3/2006-03-01/examples-1.json:1178]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: aws_guardduty/lib/botocore/data/waf-regional/2016-11-28/examples-1.json [hunk 8]
```diff
@@ -0,0 +1,1017 @@
...
+        "title": "To get a change token"
+      }
+    ],
+    "GetChangeTokenStatus": [
+      {
+        "input": {
+          "ChangeToken": "abcd12f2-46da-4fdb-b8d5-fbd4c466928f"
+        },
+        "output": {
+          "ChangeTokenStatus": "PENDING"
```
Comment (ns-snekkar) [inline aws_guardduty/lib/botocore/data/waf-regional/2016-11-28/examples-1.json:403]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: aws_guardduty/lib/botocore/data/waf/2015-08-24/examples-1.json [hunk 9]
```diff
@@ -0,0 +1,1017 @@
...
+        "title": "To get a change token"
+      }
+    ],
+    "GetChangeTokenStatus": [
+      {
+        "input": {
+          "ChangeToken": "abcd12f2-46da-4fdb-b8d5-fbd4c466928f"
+        },
+        "output": {
+          "ChangeTokenStatus": "PENDING"
```
Comment (ns-snekkar) [inline aws_guardduty/lib/botocore/data/waf/2015-08-24/examples-1.json:403]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

File: aws_guardduty/lib/botocore/data/dms/2016-01-01/examples-1.json (hunk 1 above)
Comment (nijrajgelani-crest) [reply aws_guardduty/lib/botocore/data/dms/2016-01-01/examples-1.json:40]: This is an example ARN part of the AWS SDK library.

File: aws_guardduty/lib/botocore/data/eks/2017-11-01/examples-1.json (hunk 2 above)
Comment (nijrajgelani-crest) [reply aws_guardduty/lib/botocore/data/eks/2017-11-01/examples-1.json:9]: This is an example Token part of the AWS SDK library.

File: aws_guardduty/lib/botocore/data/fsx/2018-03-01/examples-1.json (hunk 4 above)
Comment (nijrajgelani-crest) [reply aws_guardduty/lib/botocore/data/fsx/2018-03-01/examples-1.json:170]: This is an example Token part of the AWS SDK library.

File: aws_guardduty/lib/botocore/data/importexport/2010-06-01/paginators-1.json (hunk 5 above)
Comment (nijrajgelani-crest) [reply aws_guardduty/lib/botocore/data/importexport/2010-06-01/paginators-1.json:4]: This is an example Token part of the AWS SDK library.

File: aws_guardduty/lib/botocore/data/s3/2006-03-01/examples-1.json (hunk 7 above)
Comment (nijrajgelani-crest) [reply aws_guardduty/lib/botocore/data/s3/2006-03-01/examples-1.json:1178]: This is an example Token part of the AWS SDK library.

File: aws_guardduty/lib/botocore/data/waf-regional/2016-11-28/examples-1.json (hunk 8 above)
Comment (nijrajgelani-crest) [reply aws_guardduty/lib/botocore/data/waf-regional/2016-11-28/examples-1.json:403]: This is an example Token part of the AWS SDK library.

File: aws_guardduty/lib/botocore/data/waf/2015-08-24/examples-1.json (hunk 9 above)
Comment (nijrajgelani-crest) [reply aws_guardduty/lib/botocore/data/waf/2015-08-24/examples-1.json:403]: This is an example Token part of the AWS SDK library.

**Comment by nijrajgelani-crest** [comment]
Added comments for all Cycode check failures.

**Pull request merged**